walletkit_client.go

```diff
@@ -386,3 +392,25 @@ func (m *walletKitClient) BumpFee(ctx context.Context, op wire.OutPoint,
 	)
 	return err
 }
+
+// ListAccounts retrieves all accounts belonging to the wallet by default.
+// Optional name and addressType can be provided to filter through all of the
+// wallet accounts and return only those matching.
+func (m *walletKitClient) ListAccounts(ctx context.Context, name string,
+	addressType walletrpc.AddressType) ([]*walletrpc.Account, error) {
+
+	rpcCtx, cancel := context.WithTimeout(ctx, m.timeout)
+	defer cancel()
+
+	resp, err := m.client.ListAccounts(rpcCtx,
```

Comment:
nit: break before `rpcCtx`.

Reply:
Ah yes, of course.

Reply:
fixed your comment and added `.WithMacaroonAuth`